sdks/java/core/src/main/java/org/apache/beam/sdk/options/ProxyInvocationHandler.java

```diff
@@ -202,7 +204,7 @@ static BoundValue fromDefault(@Nullable Object value) {
    */
   synchronized <T extends PipelineOptions> T as(Class<T> iface) {
     checkNotNull(iface);
-    checkArgument(iface.isInterface());
+    checkArgument(iface.isInterface(), "Not an interface: %s", iface.toString());
```

Comment:
no `toString()` in Guava `checkX` methods, it will happen automatically and only if the check fails.